examples/vm-alpine-datavolume.yaml

```diff
@@ -21,7 +21,8 @@ spec:
       source:
         http:
           url: http://cdi-http-import-server.kubevirt/images/alpine.iso
-    status: {}
+    status:
+      restartCount: 0
```

Comment:
this is strange, users shouldn't (and probably can't) set status values during create/update. 

Did something change with optional vs. required here?

Reply:
Yeah this is unfortunate, someone forgot to add optional to `restartCount` which is a field that was recently added.  I will fix in CDI.  But the manifest is create-able.

Reply:
It is also okay to leave out the field when creating a datavolume, it just shows up here because the manifest is generated from code.

Reply:
i'm nervous how we treat that value in our validation webhook now though.

For example, if someone uses the DataVolumeTemplates section and doesn't set the restartCount value in the status, our validation webhook might reject the VM entirely because we're generating our logic based on this import.

our functional tests wouldn't catch this unless we're posting a VM with a DataVolumeTemplates section from yaml, which we're not

Reply:
`make cluster-up` followed by `k apply -f examples/vm-alpine-datavolume.yaml` works fine.

Even if this had the `omitempty` tag (and restartCount was excluded from the yaml), when deserialized, the object in memory would be exactly the same.